storybook/stories/core/components/ImageCrop/ImageCrop.stories.js

```diff
@@ -0,0 +1,74 @@
+import ImageCropper from '@orangehrm/oxd/core/components/ImageCropper/ImageCropper';
+
+export default {
+  title: 'Information/ImageCropper',
```

Comment:
The story should not be there under 'Information' group. Move into the correct group.

Reply:
Profile Component also inside the information group , might be needed to create new category , will check